deploy.py

```diff
@@ -13,105 +13,86 @@ def add_cli_args():
                                  "openlab-ha"],
                         help='OpenLab deployment type',
                         )
+    parser.add_argument('--action',
+                        choices=["deploy", "new-slave", "switch-role",
+                                 "show-graph", "show-ip"],
+                        default="deploy",
+                        help="The action that labkeeper supports.\n"
+                             "'deploy': create a new OpenLab CI environment.\n"
+                             "'new_slave': create new slave nodes base on "
+                             "current master nodes.\n"
+                             "'switch_role': switch the master/slave role of "
+                             "current deployment. This will re-config the "
+                             "existing OpenLab environment.\n"
+                             "'show_graph': show the hosts graph of ansible "
+                             "inventory.\n"
+                             "'show_ip': show hosts ip address.\n")
     parser.add_argument('-u', '--user',
                         help='the Ansible remote user performing deployment, '
                              'default is "ubuntu" configured in ansible.cfg',
                         )
-    parser.add_argument('--switch-role',
-                        action='store_true',
-                        help='switch the master/slave role of current deployment, '
-                             'this will re-config an existing OpenLab environment to '
-                             'switch the "master/slave" role of hosts',
-                        )
-    parser.add_argument('--new-slave',
-                        action='store_true',
-                        help='deploy the slave part of a CI cluster freshly, you need '
-                             'to specify the new slave hosts IP addresses with "--new-ip" '
-                             'argument, for example: '
-                             '--new-ip zuul02=192.168.5.5 --new-ip nodepool02=192.168.6.6',
-                        )
     parser.add_argument('-e', '--extra-vars',
                         metavar="<key=value>",
                         action='append',
                         default=[],
                         help='extra vars passed to Ansible command',
                         )
-    parser.add_argument('--new-ip',
+    parser.add_argument('--host-ip',
                         metavar="<key=value>",
                         action='append',
-                        help='specify new ip address for current inventory, this argument '
-                             'need to use with key-value pairs combined with "=", the key '
-                             'must be the host name defined in the inventory yaml files, '
-                             'e.g. zuul01, nodepool02, allinone01.',
-                        )
-    parser.add_argument('--with-vars',
-                        action='store_true',
```

Comment:
why remove this ?

Reply:
Not sure it's useful, I can add it back if there is some use cases?

Reply:
it is useful I think, if you want to confirm some inventory config options before run deployment, this can show all the hostvars based on current inventory

Reply:
Done